source/dotnet/Library/AdaptiveCards/docs/AdaptiveCards.md

```diff
@@ -2673,7 +2673,7 @@ Properties which control rendering of input labels
 
 ##### Summary
 
-Specifies the spacing bewteen the label and the input
+Specifies the spacing between the label and the input
```

Comment:
@paulcam206 i was wanting to add in specifics about the default values also for these- is that just manual text at this point or some magic metadata being read someplace?